### PR TITLE
noVNC Mobile

### PR DESCRIPTION
**Pull request by mkunz7**
updated old pr to latest

### CODE REVIEW

**Comment by mkunz7** [comment]
old pr https://github.com/sandia-minimega/minimega/pull/859

File: misc/web/novnc/include/ui.js [hunk 1]
```diff
@@ -1260,7 +1260,7 @@ var UI;
 
         // Display the desktop name in the document title
         updateDocumentTitle: function(rfb, name) {
-            document.title = name + " - noVNC";
+            //document.title = name + " - noVNC";
```
Comment (floren) [inline misc/web/novnc/include/ui.js]: what's up with this?

**Comment by floren** [comment]
Tested, seems fine. @djfritz @jcrussell any complaints?

File: misc/web/novnc/include/ui.js (hunk 1 above)
Comment (mkunz7) [inline misc/web/novnc/include/ui.js]: vnc_auto.html sets the window title to the vm name, iirc this gets overwritten when calling UI.connect() so I commented out the update

**Pull request merged**